src/utils/__tests__/paths.spec.ts

```diff
@@ -25,31 +25,43 @@ describe('paths', () => {
   const cases: Record<keyof Paths, expectedData> = {
     appHome: {
       win32:  '%APPDATA%/rancher-desktop/',
+      linux:  '%HOME%/.config/rancher-desktop/',
       darwin: '%HOME%/Library/Application Support/rancher-desktop/',
     },
+    altAppHome: {
+      win32:  '%APPDATA%/rancher-desktop/',
+      linux:  '%HOME%/.config/rancher-desktop/',
```

Comment:
Shouldn't the Linux location also be `%HOME%/.rd/`?

Reply:
It is just for consistency. When you talk to people on Slack, it is not always clear which platform they are on. So having the same answer for Linux and macOS is helpful. Also for docs, if it makes it into docs.

The same reason I want to be able to say `rdctl shell cat /etc/os-release` and have it work everywhere.